README.md

```diff
@@ -0,0 +1,104 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+Welcome to Apache OpenNLP!
+===========
+
+[![Build Status](https://github.com/apache/opennlp-addons/workflows/Java%20CI/badge.svg)](https://github.com/apache/opennlp-addons/actions)
+[![Contributors](https://img.shields.io/github/contributors/apache/opennlp-addons)](https://github.com/apache/opennlp-addons/graphs/contributors)
+[![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/apache/opennlp-addons.svg)](https://github.com/apache/opennlp-addons/pulls)
+[![Stack Overflow](https://img.shields.io/badge/stack%20overflow-opennlp-f1eefe.svg)](https://stackoverflow.com/questions/tagged/opennlp)
+
+The Apache OpenNLP library is a machine learning based toolkit for the processing of natural language text.
+
+These addons of the toolkit are written in Java and provide components for special NLP tasks, such as 
+geographic entity linking, Wordnet dictionary access and integration with Liblinear and Morfologik!
+
+The goal of the OpenNLP addons is to provide extra components, potentially in an experimental stage.
+
+OpenNLP addons code can be used both programmatically through its Java API, some components even from a terminal through its CLI.
+
+## Useful Links
+
+For additional information, visit the [OpenNLP Home Page](http://opennlp.apache.org/)
```

Comment:
https to avoid a redirect? If so there are other http URLs here too...

Reply:
nice catch, will also fix for other occurrences.